File: problems/a/areal/areal.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""
from math import sqrt


def main():
    area = int(input())
    fence_length = sqrt(area) * 4

    print(fence_length)


if __name__ == '__main__':
    main()

File: problems/a/areal/areal.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>
#include <cmath>
#include <iomanip>

using namespace std;

int main() {
    double area, fence_length;

    cin >> area;

    fence_length = 4.0 * sqrt(area);

    cout << fixed;
    cout << setprecision(8);
    cout << fence_length << endl;
}